rest-api-spec/src/main/resources/rest-api-spec/api/inference.put_elser.json

```diff
@@ -0,0 +1,35 @@
+{
+  "inference.put_elser": {
```

Comment:
This service is deprecated. I'm not sure if there's something we can add here to indicate that

Reply:
Thanks! Fixed in ac800cbf3c80c2ae89a6f66bd0eef0c27c8df5bc